packages/graphiql/resources/renderExample.js

```diff
@@ -72,20 +91,26 @@ function updateURL() {
 // Defines a GraphQL fetcher using the fetch API. You're not required to
 // use fetch, and could instead implement graphQLFetcher however you like,
 // as long as it returns a Promise or Observable.
-function graphQLFetcher(graphQLParams) {
+function graphQLFetcher(graphQLParams, headers = {}) {
```

Comment:
Hmm. I think we should make the second arg an object with the following type ` { headers: {[key:string]: any}`

Reply:
The reason being that not all fetchers will care about headers and maybe instead would want other kinds of properties.

Reply:
@connorshea do you think you could add this to the PR? makes perfect sense, allows us to pass in other values, if we continue with this pattern in 2.0.0